src/buttons/ActionLink.tsx

```diff
@@ -83,7 +83,8 @@ interface ActionLinkProps {
   className?: string;
   allowDefault?: boolean;
   label?: string;
-  isCurrent?: boolean | "time" | "false" | "true" | "page" | "step" | "location" | "date" | undefined;
+  //This allows aria-current to be set as appropriate
```

Comment:
Just "Allows aria-current to be set as appropriate".

Also space between `//` and the sentence.

Reply:
Done